syntax = "proto3";

import "trade/vo/OrderVo.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "TradeQueryHistoryDeliverListResponseProto";
message TradeQueryHistoryDeliverListResponse {
	repeated OrderVo data = 1;
}
